### DISCUSSION TITLE
Example service is down

### DISCUSSION THREAD

**Discussion by Dunkhan** [mat/besticon#77] [General]
I assume this is because heruko is no longer offering their free tier. I looked around for my own projects and the hosting provider render.com has a free tier and supports docker.

**Comment by photonometric** [comment]
@Dunkhan nice, render.com looks like a good option. The free tier for static site or services has 100GB traffic/month. The only downside is that free-tier services are put in a suspended state if inactive for 15 minutes; a request made to a service in this state can take up to 30 seconds, while the service re-initializes. 

[https://render.com/docs/free#free-web-services](https://render.com/docs/free#free-web-services)
